src/components/KMethodBadge/KMethodBadge.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <KBadge
+    :background-color="methodBadgeColors[method].backgroundColor"
+    :border-color="methodBadgeColors[method].borderColor"
+    class="k-method-badge"
+    :class="[ `method-${method}`, { 'k-method-toggle': isToggle }]"
+    :color="methodBadgeColors[method].color"
+    :is-bordered="isToggle"
+    max-width="auto"
```

Comment:
question: you have `max-width: auto` here; however, aren't all the method badges the same width? (looking at the new design [here](https://www.figma.com/file/Yze0SWXl5nKjR0rFdilljK/Components?type=design&node-id=467-10283&t=taJN6RPvzvewyenM-0))

Reply:
`max-width="auto"` just prevents it from truncating. Now in the link you sent the square ones are all same width - do we think they all (both square and rounded) should be same width? In my opinion either both should be same width or none - we don't want round badges to be content width while square ones are fixed.

Reply:
I'm good with whatever you think here; just wanted to bring it up

Reply:
Okay, I guess I wanna leave it flexible in this component for now - if we need them fixed width we can either change it later or control that from the host app

Reply:
+1 on leaving it until we implement the new designs everywhere